README.md

```diff
@@ -11,3 +11,20 @@ Kubernetes. This specific project will include:
 * Writing a report that summarizes the different approaches that can be used for
   topology aware routing along with a recommendation.
 
+## Usage
+go run main.go -input=inputFile -output=outputFile -alg=algorithm
+
+example of intput file (csv): each zone with number of nodes first, number of endpoints next  
+input name, zone1, zone2, zone3  
+perfect input, 10 10, 10 10, 20 20
```

Comment:
This should be a code block - add lines before and after with 3 backticks (```)

Reply:
fixed